app/models/conversion_host.rb

```diff
@@ -21,31 +22,81 @@ class ConversionHost < ApplicationRecord
 
   include_concern 'Configurations'
 
-  # To be eligible, a conversion host must have the following properties
-  #  - A transport mechanism is configured for source (set by 3rd party)
-  #  - Credentials are set on the resource and SSH connection works
-  #  - The number of concurrent tasks has not reached the limit
+  # Check using all associated authentications if any are directly associated
+  # with the conversion host. Otherwise, use the default check which uses the
+  # associated resource's authentications.
+  #
+  # This method is necessary to comply with AuthenticationMixin interface.
+  #
+  def verify_credentials(_auth_type = nil, _options = {})
```

Comment:
Hm we already have a common `verify_credentials_with_ssh` method for hosts [`Host#verify_credentials_with_ssh`](https://github.com/ManageIQ/manageiq/blob/master/app/models/host.rb#L769-L789), is that method something that you could use?  We could move it into a common place like the `AuthenticationMixin`

Reply:
@agrare In theory yes, though I think there are some implementation details in there that might not be right for conversion hosts, e.g. I'm not sure we're pinned to Linux vms.

Reply:
I'm not sure that check was for anything else than failing fast for hosts that we don't expect to even have the ssh daemon?  We could refactor that method if we need.

Reply:
Also, are we expecting people to deploy conversion hosts on BSD frequently? :laughing:

Reply:
I dunno, maybe even Solaris? Didn't want to make any assumptions. :)

Reply:
@agrare I think I'd like to hold off on that refactoring, in part because I think MiqSshUtil is overworked for just testing a connection. Looks to me like it's meant for cases where we actually care about data, and here we don't.